buildSrc/subprojects/binary-compatibility/src/main/groovy/gradlebuild/binarycompatibility/rules/AbstractGradleViolationRule.groovy

```diff
@@ -142,17 +142,29 @@ abstract class AbstractGradleViolationRule extends AbstractContextAwareViolation
             change.changes
         )
 
-        def id = "accept" + (change.type + change.member).replaceAll('[^a-zA-Z0-9]', '_')
+        def changeId = (change.type + change.member).replaceAll('[^a-zA-Z0-9]', '_')
         Violation violation = Violation.error(
             member,
             rejection.getHumanExplanation() + """. If you did this intentionally, please accept the change and provide an explanation:
-                <a class="btn btn-info" role="button" data-toggle="collapse" href="#${id}" aria-expanded="false" aria-controls="collapseExample">Accept this change</a>
-                <div class="collapse" id="${id}">
+                <a class="btn btn-info" role="button" data-toggle="collapse" href="#accept-${changeId}" aria-expanded="false" aria-controls="collapseExample">Accept this change</a>
+                <div class="collapse" id="accept-${changeId}">
                   <div class="well">
                       In order to accept this change add the following to <code>subprojects/architecture-test/src/changes/accepted-public-api-changes.json</code>:
                     <pre>${prettyPrintJson(acceptanceJson)}</pre>
                   </div>
-                </div>""".stripIndent()
+                </div>
+                <p>
+                Or update baseline version:
```

Comment:
There are three things you may need to do:
- Accept the change
- Merge `release` to `master`
- Publish a nightly from `release`

We should have some documentation here how to decide which of the things to do. So we probably need three buttons? We can also try to come up with some message together.